frontend-react/src/components/annotate.js

```diff
@@ -41,17 +42,30 @@ class Annotate extends Component {
         }
 
         return (
-            <div className="backgroundAll">
+            <div className="border">
                 <form className="selectForm" onSubmit={this._handleSubmit}>
                     <input type="file" id="file" className="fileInputButton" onChange={this._handleImageChange} />
                     <label for="file" class="fileLabel">Select Image</label>
```

Comment:
```suggestion
                    <label htmlFor="file" className="fileLabel">Select Image</label>
```
These need to be changed to suit the React naming style.